src/Components/Footer.js

```diff
@@ -1,20 +1,28 @@
 import React from "react";
-import Typography from '@material-ui/core/Typography';
-import {Link} from "react-router-dom";
+import Typography from "@material-ui/core/Typography";
+import { Link } from "react-router-dom";
 
 function Footer() {
   return (
-    <div>  
-      <span  style={{float : 'left'}}>
-    <Typography variant="h6">
-      <a style={{textDecoration: 'none'}} href="https://github.com/Strategizer/app">Open Source</a>
-    </Typography></span>
-      <span  style={{float : 'right'}}>
-    <Typography variant="h6">
-      <Link style={{textDecoration: 'none'}} to="/contributors">Contributors</Link>
-    </Typography></span>
+    <div>
+      <span style={{ float: "left" }}>
+        <Typography variant="h6">
+          <a
+            style={{ textDecoration: "none" }}
+            href="https://github.com/Strategizer/app"
+          >
+            Open Source
+          </a>
+        </Typography>
+      </span>
+      {
+        //<span  style={{float : 'right'}}>
```

Comment:
If just commenting this part is the only change in this file then please revert it (this file shouldn't be in the PR) as there is already an issue created for Footer that is already assigned to someone

Reply:
Ooops.. I took that out to check the placement of the Contributors section of the header.. meant to put it back in.. fixing now